app/constants/filename-regex.js

```diff
@@ -0,0 +1,3 @@
+const filenameRegex = /^\w+\.pdf$/
```

Comment:
Should we make this more restrictive?

PDFs are in format FFC_PaymentStatement_SFI_2022_<frn>_<timestamp>.pdf

Could be extended for other schemes.

Reply:
We're shortly going to have payment schedules too so if we were I'd suggest we create a regex that supports that too.

But do we need to restrict the API in this way.  As long as they're meeting the requirements of the URL and provide a filename they'll just get a 404 if that file doesn't exist.